src/main/scala/io/iohk/ethereum/jsonrpc/EthService.scala

```diff
@@ -264,6 +276,42 @@ class EthService(
     }
   }
 
+  def getCode(req: GetCodeRequest): ServiceResponse[GetCodeResponse] = {
+    Future.successful {
+      resolveBlock(req.block).map { block =>
+        val world = InMemoryWorldStateProxy(blockchainStorages, Some(block.header.stateRoot))
```

Comment:
> I think these additional features don't cause any overhead when just calling getCode, do they?

They induce additional level of coupling which may increase the later effort of separating the Ledger or the VM.